sdks/python/container/Dockerfile

```diff
@@ -101,7 +101,7 @@ RUN rm /opt/apache/beam/third_party_licenses/golang/LICENSE
 
 COPY target/license_scripts /tmp/license_scripts/
 RUN if [ "$pull_licenses" = "true" ] ; then \
-      pip install 'pip-licenses<4.0.0' pyyaml tenacity && \
+      pip install 'pip-licenses<=4.1.0' pyyaml tenacity && \
```

Comment:
using <= in version constraints is almost always wrong, as that excludes newer minor version, and is largely equivalent to ==4.1.0.

Reply:
Ah. I started with `<5` but for some reason I changed it to `<=`. committed the updated code now.